sklearn/metrics/tests/test_score_objects.py

```diff
@@ -247,6 +247,31 @@ def test_regression_scorers():
     assert_almost_equal(score1, score2)
 
 
+def test_log_loss():
```

Comment:
Please change the name of the test to something more specific, we are not testing log_loss but the correctness of the labels argument.